src/content/auth.md

```diff
@@ -66,3 +66,40 @@ Next, request with "Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d" h
 curl -XGET -H 'Content-Type: application/edn' -H 'Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d' http://localhost:45102/default/jobs
 {:results [...]}
 ```
+
+## OAuth2.0 Authentication
+
+OAuth2.0 is available for authentication when you use JobStreamer.
+If you use OAuth2.0, you login as a "guest" user with operator role.
```

Comment:
If you login to JobStreamer by using OAuth2.0, You will get the "operator" role.